src/api/background/background-setup.ts

```diff
@@ -36,8 +37,8 @@ export function setupBackgroundService() {
 
   // MESSAGES:
   // Watch for API call and chunk messages:
-  onMessage("api_call", handleApiCallMessage);
-  onMessage("chunk", handleChunkMessage);
+  isomorphicOnMessage("api_call", handleApiCallMessage);
```

Comment:
Verify handlers cannot be called directly from the SDK (except for the ones that are actually meant to be called from the SDK).